CHANGELOG.md

```diff
@@ -1,204 +1,256 @@
 # Change Log
 
-## [v0.1.1](https://github.com/skippbox/kompose/tree/v0.1.1) (2016-10-06)
-[Full Changelog](https://github.com/skippbox/kompose/compare/v0.1.0...v0.1.1)
+## [v0.1.2](https://github.com/kubernetes-incubator/kompose/tree/v0.1.2) (2016-10-31)
+[Full Changelog](https://github.com/kubernetes-incubator/kompose/compare/v0.1.1...v0.1.2)
 
 **Implemented enhancements:**
```

Comment:
>Oh, ok :-) We have to start labeling all issues properly then.

Yes that's good.

>can we maybe move those 3 issues to "closed issues" section?

Yes that's pretty easy but I think we should keep the idea of classifying issues by label like this. That makes the CHANGLOG more specific. I will double-check the closed issues to see if any "enhancements" is missing. Will update shortly and let you know.